backend/server/routers/auth_utility/oidc/errors.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+from typing import Any, Dict, Optional
+
+class OIDCError(Exception):
+    # base class
+    def __init__(self, error_description: Optional[str] = None) -> None:
+        self.error_description: Optional[str] = error_description
+
+    def __str__(self) -> str:
+        desc = self.error_description or "No description."
+        return f"{super().__str__()}\n  Description: {desc}"
```

Comment:
Super nitpicky but, I woudl remove the single space after `\n` and replace it with a `\t` or, just remove the ` ` entirely

Reply:
true